### PR TITLE
fix: mse metric computation

### LINKED ISSUE #3: evaluation bug report
Hi, there
I found that the evaluation code was wrong.
https://github.com/surgical-vision/REIM-NeRF/blob/7578806b92c28cfdc41d89bb1e6c80eacb5a9f70/scripts/evaluate_c3vd.py#L148-L150
The frame_data['scene_scale'] is used to recover the gt depth range, 
i.e. the `real_gt= ref_depthmaps / frame_data['scene_scale']` 

However due to the square operation in depth_se, the final results should be
`depth_mse = torch.mean(depth_se[...,mask], dim=-1)/(frame_data['scene_scale']**2)`
the origin code missed a frame_data['scene_scale'] in the denominator so the results reported in the paper was especially small mse=0.013mm^2 (which seemed impossible)
I run a toy experiment to fixed code, which the mse turn to ~1.305mm^2( a more reasonable results)
I am sorry for any inconvience this issue caused and wish you can fixed this small bug.

### PR DESCRIPTION
**Pull request by haowang020110**
fixed mse bug

### CODE REVIEW

**Review by dimitrisPs — APPROVED** [review]
Apologies for the delayed response, I missed the notification completely. 

Thanks for fixing this
